composer.json

```
@@ -5,8 +5,8 @@
     "license": "OSL-3.0",
     "authors": [
         {
-            "name": "PrestaShop SA",
-            "email": "contact@prestashop.com"
+            "name": "MBE Worldwide",
+            "email": "contact@mbecorporate.com/"
```

```suggestion
            "email": "contact@mbecorporate.com"
```
sorry, incorrect email.

@matks wdyt?